tests/test_cli.py

```diff
@@ -51,6 +51,23 @@ def test_version_verbose_logging():
     assert "Verbose logging is enabled." in result.output.strip()
 
 
+def test_hidden_stacktrace():
+    """
+    Arrange/Act: Run `profiles edit invalid` subcommand.
+    Assert:  The output does not contain stacktrace.
```

Comment:
Maybe I am a bit blind, but are you actually asserting this anywhere? 😆

Reply:
Haha, sorry I am not. :blush: As mentioned in OP, couldn't make the test work. The test here was just to document what I tried and also to verify with Codecov that the custom exception hook is indeed not executed. If you think the test is useful on its own, I can keep it with an updated name, otherwise I'll remove it.

Reply:
So, I researched and tried for a while to see if there is a way we can test it, but I did not find any. To my understanding, the `CliRunner` actually interferes with the exception handling and so we never get to the point that the registered exception handler is executed. I even looked into alternative implementations that might mitigate the issue, e.g., sub-classing the `cli.group()` class like [here](https://github.com/pallets/click/issues/1547), however then I had problems obtaining the value for the verbose option.

Since the test coverage is only slightly negatively impacted and all other tests appear to be unaffected, I suggest that we merge this _as is_ and then revisit this should we run into test issues or related bugs in the future.